test_package_installer/conanfile.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from conans import ConanFile, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch"
+
+    def build(self):
+        pass
+
+    def test(self):
+        if tools.cross_building(self.settings):
+            self.output.warn("Skipping test package: Target is incompatible with current arch.")
+        self.run("thrift --version", run_environment=True)
```

Comment:
this looks wrong to me. If it is NOT cross building we want to execute it.

https://github.com/bincrafters/conan-templates/blob/4637b29f63f53ba21f3279f20d4164332970f8a2/default/test_package/conanfile.py#L16

Reply:
you are right! I'll copy from test_package. thanks

Reply:
done!